cmd/controller/app/options/options.go

```diff
@@ -261,5 +261,9 @@ func EnabledControllers(o *config.ControllerConfiguration) sets.Set[string] {
 		enabled = enabled.Insert(shimgatewaycontroller.ControllerName)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(feature.ValidateCAA) {
+		logf.Log.Info("ValidateCAA is schedule for removal in v1.18")
+	}
```

Comment:
suggestion: Thanks for raising this PR - we discussed this this morning and agree with the timeline for this.

I'm going to propose an alternative wording; what do you think?

```suggestion
	if utilfeature.DefaultFeatureGate.Enabled(feature.ValidateCAA) {
		logf.Log.Info("the ValidateCAA feature flag is scheduled for removal in v1.18 and will become a no-op")
	}
```